kubeconfig-template.yaml

```diff
@@ -11,13 +11,19 @@ contexts:
   name: hashicorp-learn
 clusters:
 - cluster:
-    server: https://${endpoint}
+    server: ${endpoint}
     certificate-authority-data: ${cluster_ca}
   name: ${cluster_name}
 users:
 - name: ${user_name}
   user:
-    password: ${user_password}
-    username: ${user_name}
-    client-certificate-data: ${client_cert}
-    client-key-data: ${client_cert_key}
\ No newline at end of file
+    exec:
```

Comment:
Can we use the original one instead? The AWS CLI prob won't be configured/installed in the katacoda scenario... unless you're planning on updating the katacoda scenario too

Reply:
I don't think the EKS module supports user/password auth? Unless I'm totally missing something? I don't think installing the AWS CLI on the katacodas will be difficult, though.

Reply:
Oh alright, just wanted to call it out, since this uses the AWS CLI 😄